src/core/mempool.rs

```diff
@@ -979,7 +979,7 @@ impl MemPoolDB {
             sponsor_nonce,
         )?;
 
-        Ok(())
+        Ok(tx)
```

Comment:
Why is this returning `tx`?  If the caller needs the transaction after submitting it, then maybe it make more sense to just have this method take `tx: &StacksTransaction` as an argument, and continue to return `Ok(())`?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2204/commits/7e4e9dffb1600aa28cbcc38c3cd798c67c0675e2